gui/network/models.py

```diff
@@ -70,6 +70,13 @@ class GlobalConfiguration(Model):
             regex=r'^[a-zA-Z\.\-\_0-9]+$',
         )],
     )
+    gc_domains = models.TextField(
+        max_length=256,
+        verbose_name=_("Additional domains"),
+        default='',
+        blank=True,
+        help_text="Delimit domains by spaces."
```

Comment:
You're missing a translation of help_text, using _("")

Also, that text is not very descriptive. @wblock can you please help us here with something better to write here? (refer to man resolv.conf, "search" directive).